packages/orbit-components/src/Stack/Stack.stories.tsx

```diff
@@ -37,845 +38,355 @@ import ChoiceGroup from "../ChoiceGroup";
-    control: {
-      type: "select",
+    spaceAfter: {
+      options: Object.values(SPACINGS_AFTER),
+      control: {
+        type: "select",
+      },
     },
   },
 };
 
-export const MediumMobileProperties = ({
-  inline,
-  direction,
-  wrap,
-  grow,
-  shrink,
-  basis,
-  align,
-  justify,
-  spacing,
-  spaceAfter,
-}) => {
-  const mediumMobile = {
-    inline,
-    direction,
-    wrap,
-    grow,
-    shrink,
-    basis,
-    align,
-    justify,
-    spacing,
-    spaceAfter,
-  };
+export default meta;
+type Story = StoryObj<typeof Stack>;
 
-  return (
-    <Stack mediumMobile={mediumMobile}>
-      <InputField type="password" label="Password" help="You need some help!" />
-      <Button>Sign In</Button>
+export const Default: Story = {
+  render: () => (
+    <Stack>
+      <Button type="primary" iconLeft={<Airplane />}>
+        Button
+      </Button>
+      <Button type="secondary">Button</Button>
     </Stack>
-  );
-};
-
-MediumMobileProperties.story = {
-  name: "MediumMobile properties",
+  ),
 
   parameters: {
-    info: "You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
+    info: "This is the default configuration of this component. Visit Orbit.Kiwi for more detailed guidelines.",
+    controls: {
+      disable: true,
+    },
   },
 };
 
-MediumMobileProperties.args = {
-  inline: false,
-  direction: undefined,
-  wrap: false,
-  grow: true,
-  shrink: false,
-  basis: "auto",
-  align: ALIGNS.END,
-  justify: undefined,
-  spacing: undefined,
-  spaceAfter: undefined,
-};
+export const MobileProperties: Story = {
+  render: args => (
+    <Stack {...args}>
+      <Button type="primary" iconLeft={<Airplane />}>
+        Button
+      </Button>
+      <Button type="secondary">Button</Button>
+    </Stack>
+  ),
 
-MediumMobileProperties.argTypes = {
-  direction: {
-    options: Object.values(DIRECTIONS),
-    control: {
-      type: "select",
-    },
-  },
-  align: {
-    options: Object.values(ALIGNS),
-    control: {
-      type: "select",
-    },
-  },
-  justify: {
-    options: Object.values(JUSTIFY),
-    control: {
-      type: "select",
-    },
-  },
-  spacing: {
-    options: Object.values(SPACINGS),
-    control: {
-      type: "select",
-    },
-  },
-  spaceAfter: {
-    options: Object.values(SPACINGS_AFTER),
-    control: {
-      type: "select",
-    },
+  args: {
+    direction: DIRECTIONS.ROW,
+    align: ALIGNS.START,
+    justify: JUSTIFY.START,
+    spacing: SPACINGS.NONE,
   },
 };
 
-export const LargeMobileProperties = ({
-  inline,
-  direction,
-  wrap,
-  grow,
-  shrink,
-  basis,
-  align,
-  justify,
-  spacing,
-  spaceAfter,
-}) => {
-  const largeMobile = {
-    inline,
-    direction,
-    wrap,
-    grow,
-    shrink,
-    basis,
-    align,
-    justify,
-    spacing,
-    spaceAfter,
-  };
-
-  return (
-    <Stack largeMobile={largeMobile}>
+export const MediumMobileProperties: Story = {
```

Comment:
Have you considered to squash these 'viewport' stories?
The code is redundant, I was thinking about the solution, and I found something like this:

```
export const MediumMobileProperties: Story &
  StoryObj<{
    largeDesktop: boolean;
    desktop: boolean;
    tablet: boolean;
    largeMobile: boolean;
    mediumMobile: boolean;
  }> = {
  render: ({ largeDesktop, desktop, tablet, largeMobile, mediumMobile, ...args }) => {

    return (
      <Stack
        largeDesktop={largeDesktop && { ...args }}
        desktop={desktop && { ...args }}
        tablet={tablet && { ...args }}
        mediumMobile={mediumMobile && { ...args }}
        largeMobile={largeMobile && { ...args }}
        {...args}
      >
        ...
      </Stack>
    );
  },

  args: {
    direction: DIRECTIONS.COLUMN,
    largeDesktop: false,
    desktop: false,
    tablet: false,
    largeMobile: false,
    mediumMobile: false,
  },
};
```

but the "final" code in SB might look messy for the user. :(

Reply:
No I haven't consider squashing them. If we do it like this, then you cannot set those values separately for `mediumMobile`, `tablet` etc. as they all are using the same value as the default mobile properties so the only added value is that you can see it in the code source. And for this reason we didn't include those `mediumMobile`, `tablet` etc. in the Playground stories in some other batch.

Maybe it's redundant to have all those stories and the Mobile one would be sufficient, tbh I don't know anymore 😅

Reply:
yes, yes. I was also thinking about it and I haven't found any reasonable solution.
At first, my thoughts went to `Hide` component stories, but there is slightly different approach.

I would keep it as it is. 👍